whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java

```diff
@@ -233,13 +234,14 @@ private RpslObject getUpdatedObject(final Update update, final UpdateContext upd
         return updatedObject;
     }
 
-    private void validateChanged(final RpslObject updatedObject, final ObjectMessages objectMessages) {
+    private void validateChanged(RpslObject updatedObject, final ObjectMessages objectMessages) {
```

Comment:
Can you investigate moving this code to a separate class, implementing the TransformPipeline interface? Rather than modifying the updatedObject in a "validate" method.

Reply:
Also, please replace TransformPipeline.java with a loop in SingleUpdateHandler, similar to the attributeGenerators.

Reply:
Remove the Ignore annotations on the new tests once your changes are complete